experimental/traffic-portal/src/app/core/topologies/topology-details/topology-details.component.html

```diff
@@ -0,0 +1,45 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<mat-card appearance="outlined" class="page-content single-column">
+	<tp-loading *ngIf="!topology"></tp-loading>
+	<form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="topology">
+		<mat-card-content class="container">
+			<mat-form-field>
+				<mat-label>Name</mat-label>
+				<input matInput type="text" name="name" required readonly disabled [(ngModel)]="topology.name">
```

Comment:
a two-way binding is useless on a readonly control; should bind to `[value]` instead of `[(ngModel)]`

Reply:
Using `[value]` now